episodes/project-management.md

```diff
@@ -23,83 +23,55 @@ After completing this episode, participants should be able to...
-- **Labels**: if you convert each card into an issue, then you can label them with their priority - remember GitHub lets you create custom labels and set their colours. Label colours can provide a very visually clear indication of issue priority but require more administrative work on the project, as each card has to be an issue to be assigned a label. If you choose this route for issue prioritisation - be aware of accessibility issues for colour-blind people when picking colours.
+> *"What is remaining to finish off this task?"*
 
-::: callout
+A Project answers the question:
 
-## Project Boards vs Milestones
-For a good description of differences between the two, have a look at [this answer on StackOverflow](https://stackoverflow.com/questions/39591795/what-is-the-difference-relationship-between-github-projects-and-milestones).
+> *"What (tasks) are we working on at the moment?"*
 
-A Project Board answers the question:
+:::
 
-> *"What are we working on at the moment?"*
+Milestones are good for managing time-bound, structured iterations and working in short sprints on batches of issues. Dedicated sprints seem to work well for lesson development, so we will have a look at how Milestones can help with this. We recommend you to also have a look at [GitHub's documentation on Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects)  and see if they are suitable for your lesson development workflow.
 
-A Milestone answers the question:
+### GitHub's Milestones
 
-> *"What is remaining to finish off this task?"*
+A Milestone is a collection of issues to be worked on in a given time period. Milestones are good for
+tracking progress as they provide an ordered list of issues which must all be resolved for the overall task to be
+considered complete. Milestones work like labels - they mark and group issues that are expected to be closed at
+some point in time.
 
-:::
+The `Milestones` page in GitHub is accessible from the `Issues` or `Pull requests` pages -
+from there you can create a new Milestone, set its description and due date, sort all Milestones by due date and see the percentage of issues completed for your Milestones.
+Within an individual Milestone's page, you can see the list of associated issues and prioritise them by pointing to the left of an issue until the 'hand' pointer appears and then drag-and-dropping it to a new location in the list (with higher position in the list indicating higher priority). Back on the `Issues`/`Pull requests` page, you can assign issues/pull requests to a Milestone and filter your issues/pull requests by Milestones.
```

Comment:
Oh cool, I did not know about the prioritisation! Good timing, too: I will do a bit of this before we point our trainee Trainers at the beta release milestone tomorrow...

Reply:
Think it is superficial a bit - you can reorder the issues and assign meaning to the order (or not). I also learned that when I was doing this PR - the GH interface changes more than I can keep up with it 😆 .